app.py

```diff
@@ -112,3 +112,17 @@ def specific_education(education_id):
         data["skill"] = data["skill"][:index][index+1:]
         return jsonify({"inf0": "Education entry {id} has been deleted"}), 204
     return jsonify({})
+
+
+@app.route('/check-spelling', methods=['POST'])
+def check_spellings():
+    '''
+    Handles spelling requests
+    '''
+    if request.method == 'POST':
+        request_data = request.get_json()
+
+        sentence = request_data['sentence']
```

Comment:
What if the 'sentence' field is not included in the request?

Reply:
I have added a fix for that by adding a try catch block for the `KeyError` exception.